common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -861,6 +862,15 @@ def student_view_data(self, context=None):
                     "file_size": 0,  # File size is unknown for fallback URLs
                 }
 
+        # Include youtube link if there is no encoding for mobile- ie only a fallback URL or no encodings at all
+        # We are including a fallback URL for older versions of the mobile app that don't handle Youtube urls
+        if self.youtube_id_1_0 and "mobile_low" not in encoded_videos:
```

Comment:
We could do that but then we would need the same check to decide whether we need a Fallback URL, right?